src/main/kotlin/com/epam/carapi/dto/ApiError.kt

```diff
@@ -0,0 +1,10 @@
+package com.epam.carapi.dto
+
+import java.time.LocalDateTime
+
+data class ApiError(
```

Comment:
I think `ApiErrorMessage` class name will be more understandable as `Error` postfix is usually used to define JVM related errors.